docs/use cases/README.md

```diff
@@ -624,3 +657,156 @@ stop;
+| РЕЗУЛЬТАТ                              | Завдання створене                                                                                                                                                                     |
+| ВИКЛЮЧНІ СИТУАЦІЇ                      | <ul><li>`CreateTask_AccessDeniedEXC`<br>учасник проєкту не має прав на створення завдання</li><li>`CreateTask_NoReferenceEXC`<br>учасник проєкту не заповнив необхідні поля</li></ul> |
+
+<div style="
+    border-radius:4px;
+    border: 1px solid #cfd7e6;
+    padding: 1em;">
+
+@startuml
+
+|#e6feff|Учасник проєкту|
+start;
+:Натискає кнопку "Створити завдання";
+
+|#e6f3ff|Система|
+:Перевіряє права учасника ;
+
+note right #ffb3b3
+CreateTask_AccessDeniedEXC
+end note
+
+:Виводить форму для створення завдання;
+
+|Учасник проєкту|
+:Заповнює форму;
+:Натискає кнопку "Створити";
+
+|Система|
+:Перевіряє наявність обовʼязкових полів;
+
+note right #ffb3b3
+CreateTask_NoReferenceEXC
+end note
+
+:Створює і відображає завдання \nу відповідній категорії;
+:Надсилає всім призначеним повідомлення \nна виконання завдання;
+
+|Учасник проєкту|
+:Завершує взаємодію з системою;
+stop;
+
+@enduml
+
+</div>
+
+### Редагувати завдання
+
+| <div style="text-align: left">ID</div> | <div style="text-align: left">`EditTask`</div>                                                                                                                                      |
+|----------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| НАЗВА                                  | Редагувати завдання                                                                                                                                                                 |
+| УЧАСНИКИ                               | Учасник проєкту, система                                                                                                                                                            |
+| ПЕРЕДУМОВИ                             | <ul><li>Учасник проєкту авторизований</li><li>Учасник проєкту обрав завдання</li></ul>                                                                                              |
+| РЕЗУЛЬТАТ                              | Завдання відредаговане                                                                                                                                                              |
+| ВИКЛЮЧНІ СИТУАЦІЇ                      | <ul><li>`EditTask_AccessDeniedEXC`<br>учасник проєкту не має прав на редагування завдання</li><li>`EditTask_NoReferenceEXC`<br>учасник проєкту не заповнив необхідні поля</li></ul> |
```

Comment:
`EditTask_MissingDataEXC` and `EditTask_InvalidDataEXC` like for the previous use case, delete `EditTask_NoReferenceEXC`